docs/setup.rst

```diff
@@ -0,0 +1,99 @@
+Install the module in Odoo
+==========================
+If you already have an Odoo instance up and running, your preferred way to install
+addons will work with `MIS Builder`.
+
+Using git
+*********
+The most common way to install the module is to clone the git repository in your
+server and add the directory to your odoo.conf:
+
+#. Clone the git repository
+
+   .. code-block:: sh
+
+      cd your-addons-path
+      git clone https://github.com/OCA/mis-builder
+      cd mis-builder
+      git checkout 10.0 #for the version 10.0
+
+#. Update the addon path of `odoo.conf`
+#. Restart Odoo
+#. Update the addons list in your database
+#. Install the MIS Builder application.
+
+Using pip
+*********
+An easy way to install it with all its dependencies is using pip:
+
+#. Recover the code from pip repository
+
+   .. code-block:: sh
+
+      pip install odoo10-addon-mis_builder odoo-autodiscover
```

Comment:
add `--pre` to be sure to get the last `.devN` version. That's still a part that annoys me that we cannot control precisly which version goes to pypi. Anyway, that's another other story.